media/js/firefox/family-nav.js

```diff
@@ -192,6 +199,11 @@ if (typeof window.Mozilla === 'undefined') {
             } else {
                 // if matchMedia not available, just wire up the desktop stuff
                 _enableDesktop();
+
+                // check if IE 8 and replace ... button
+                if (/MSIE\s[1-8]\./.test(navigator.userAgent)) {
```

Comment:
I can't decide if this check is necessary, or if the lack of `matchMedia` is enough.